library/OrderCloud.Catalyst/Auth/WebhookAuth/README.md

```diff
@@ -0,0 +1,43 @@
+## Webhook Authentication Attribute
+
+## Purpose 
+The Webhook authentication attribute ensures that the webhook request came from ordercloud. This prevents other applications from calling your webhook route directly.
+
+## How it works
+When you create a webhook in OrderCloud you are required to enter a "HashKey" (or "Secret"). 
+When OrderCloud receives a request and then sends a subsequent webhook it includes this hashed string in a header named "X-oc-hash". When you instantiate your Webhook Authentication in your application you will need to pass in your HashKey so it can be used to compare to what is received in the "X-oc-hash" header in webhook requests. If the strings match then the request is authenticated and allowed to proceed. If they do not (or the header is not present) the request will be rejected.
+
```

Comment:
> so it can be used to compare to what is received in the "X-oc-hash" header in webhook requests

This isn't entirely correct. `X-oc-hash` is actually the body hashed **with** the the stored "HashKey". Todd has a description here that we might want to borrow from a little  https://github.com/ordercloud-api/ordercloud-dotnet-sdk-extensions#ordercloud-webhook-authentication